src/community/oseo/oseo-core/src/test/java/org/geoserver/opensearch/eo/store/GSFixtureUtilitiesDelegate.java

```diff
@@ -31,7 +31,7 @@ public static Properties loadProperties(File file) {
      * ".geoserver" in the user home directory.
      */
     public static File getFixtureDirectory() {
-        return new File(System.getProperty("user.home") + File.separator + ".geoserver");
+        return new File(System.getProperty("user.home") + File.separator + "geoserver");
```

Comment:
No, it must be in ".geoserver" just like all other test fixtures, the pattern is "$home/.$projectName", so for example, we have a lot in ".geotools" too.

Reply:
fixed